<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseSurface</class>
 <widget class="QWidget" name="chooseSurface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#chooseSurface{
border-image: url(:/images/rule_background.png);
}</string>
  </property>
  <widget class="QToolButton" name="help">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>30</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;隶书&quot;;
color:white;
border:2px solid rgb(223, 223, 111);
background-color: rgba(255, 255, 254, 20);
border-radius:25px</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="rule">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>100</y>
     <width>551</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 254,30);
border-radius:25px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>100</y>
     <width>551</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:25px;
border:3px solid rgb(203, 203, 101)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 254, 10);
border:3px solid rgb(203, 203, 101);
border-radius:25px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 254, 10);
border:3px solid rgb(203, 203, 101);
border-radius:25px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>430</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 254, 10);
border:3px solid rgb(203, 203, 101);
border-radius:25px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 254, 10);
border:3px solid rgb(203, 203, 101);
border-radius:25px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="img1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius:15px;
background-color: rgba(255, 255, 254, 50);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="img2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 254, 50);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="img3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>450</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 254, 50);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="img4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>450</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 254, 50);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="sequence">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>730</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;隶书&quot;;
color:white;
border:2px solid rgb(223, 223, 111);
background-color: rgba(255, 255, 254, 20);
border-radius:25px</string>
   </property>
   <property name="text">
    <string>自定义玩家顺序</string>
   </property>
  </widget>
  <widget class="QToolButton" name="start">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>20</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;隶书&quot;;
color:white;
border:2px solid rgb(223, 223, 111);
background-color: rgba(255, 255, 254, 20);
border-radius:25px</string>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;隶书&quot;;
color:white;
border:2px solid rgb(223, 223, 111);
background-color: rgba(255, 255, 254, 20);
border-radius:25px</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'隶书'; font-size:24pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;更改玩家个数&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="subtract">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30pt &quot;Bodoni MT&quot;;
color:white;
border-radius:20px;
background-color: rgba(255, 255, 254,50);
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QToolButton" name="add">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30pt &quot;Bodoni MT&quot;;
color:white;
border-radius:20px;
background-color: rgba(255, 255, 254,50);
</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/rule_background.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="name4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>660</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;隶书&quot;;
color:white;
border:2px solid rgb(223, 223, 111);
background-color: rgba(255, 255, 254, 20);
border-radius:25px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>51</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;隶书&quot;;
color:white;
border:2px solid rgb(223, 223, 111);
background-color: rgba(255, 255, 254, 20);
border-radius:25px</string>
   </property>
   <property name="text">
    <string>请
在
头
像
下
方
的
框
中
输
入
昵
称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;隶书&quot;;
color:white;
border:2px solid rgb(223, 223, 111);
background-color: rgba(255, 255, 254, 20);
border-radius:25px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="name2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;隶书&quot;;
color:white;
border:2px solid rgb(223, 223, 111);
background-color: rgba(255, 255, 254, 20);
border-radius:25px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="name3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>660</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;隶书&quot;;
color:white;
border:2px solid rgb(223, 223, 111);
background-color: rgba(255, 255, 254, 20);
border-radius:25px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="no1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 20pt &quot;Bell MT&quot;;
color:white;
border:2px solid rgb(223, 223, 111);
background-color:rgb(139, 91, 110);
border-radius:18px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="no2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 20pt &quot;Bell MT&quot;;
color:white;
border:2px solid rgb(223, 223, 111);
background-color:rgb(139, 91, 110);
border-radius:18px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="no3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 20pt &quot;Bell MT&quot;;
color:white;
border:2px solid rgb(223, 223, 111);
background-color:rgb(139, 91, 110);
border-radius:18px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="no4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 20pt &quot;Bell MT&quot;;
color:white;
border:2px solid rgb(223, 223, 111);
background-color:rgb(139, 91, 110);
border-radius:18px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="explain">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>831</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
border:2px solid rgb(223, 223, 111);
background-image: url(:/images/rule_background.png);
border-radius:25px;
font: 16pt &quot;隶书&quot;;
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'隶书'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;❀点击上方“+”或“-”更改玩家个数。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;❀点击“自定义玩家顺序”，再点击玩家头像左上角的顺序标识，可确定该玩家为先手，否则系统将随机确定先手玩家。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;❀点击玩家头像可切换头像。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;❀准备就绪后，点击“开始游戏”即可。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="haveRead">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>530</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;隶书&quot;;
color:white;
border:2px solid rgb(223, 223, 111);
background-color: rgba(255, 255, 254, 20);
border-radius:25px</string>
   </property>
   <property name="text">
    <string>已阅</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>51</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;隶书&quot;;
color:white;
border:2px solid rgb(223, 223, 111);
background-color: rgba(255, 255, 254, 20);
border-radius:25px</string>
   </property>
   <property name="text">
    <string>温
馨
提
示

有
背
景
音
乐
哦</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>help</zorder>
  <zorder>rule</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>img1</zorder>
  <zorder>img2</zorder>
  <zorder>img3</zorder>
  <zorder>img4</zorder>
  <zorder>sequence</zorder>
  <zorder>start</zorder>
  <zorder>textBrowser</zorder>
  <zorder>subtract</zorder>
  <zorder>add</zorder>
  <zorder>name4</zorder>
  <zorder>label_6</zorder>
  <zorder>name1</zorder>
  <zorder>name2</zorder>
  <zorder>name3</zorder>
  <zorder>no1</zorder>
  <zorder>no2</zorder>
  <zorder>no3</zorder>
  <zorder>no4</zorder>
  <zorder>label_7</zorder>
  <zorder>explain</zorder>
  <zorder>haveRead</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
